detection_rules/main.py

```diff
@@ -345,3 +347,58 @@ def test_rules(ctx):
 
     clear_caches()
     ctx.exit(pytest.main(["-v"]))
+
+
+@root.command("kibana-push")
+@click.pass_context
+@click.argument("local-repo", default=get_path("..", "kibana"))
+@click.option("--kibana-directory", "-d", help="Directory to overwrite in Kibana",
+              default="x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules")
+@click.option("--ssh/--http", is_flag=True, help="Method to use for cloning")
+@click.option("--github-repo", "-r", help="Repository to use for the branch", default="elastic/kibana")
+@click.option("--message", "-m", help="Override default commit message")
+def kibana_push(ctx, local_repo, github_repo, ssh, kibana_directory, message):
+    """Prep a commit and push to Kibana."""
+    git_exe = shutil.which("git")
+
+    package_name = load_dump(PACKAGE_FILE)['package']["name"]
+    release_dir = os.path.join(RELEASE_DIR, package_name)
+    message = message or f"[Detection Rules] Add {package_name} rules"
+
+    if not os.path.exists(release_dir):
+        click.secho("Release directory doesn't exist.", fg="red", err=True)
+        click.echo(f"Run {click.style('python -m detection_rules build-release', bold=True)} to populate", err=True)
+        ctx.exit(1)
+
+    if not git_exe:
+        click.secho("Unable to find git", err=True, fg="red")
+        ctx.exit(1)
+
+    try:
+        if not os.path.exists(local_repo):
+            if not click.confirm(f"Kibana repository doesn't exist at {local_repo}. Clone?"):
+                ctx.exit(1)
+
+            url = f"git@github.com:{github_repo}.git" if ssh else f"https://github.com/{github_repo}.git"
+            subprocess.check_call([git_exe, "clone", url, local_repo, "--depth", 1])
+
+        def git(*args, show_output=False):
+            method = subprocess.call if show_output else subprocess.check_output
+            return method([git_exe, "-C", local_repo] + list(args), encoding="utf-8")
+
+        git("checkout", "master")
+        git("pull")
+        git("checkout", "-b", f"rules/{package_name}", show_output=True)
+        git("rm", "-r", kibana_directory)
+
+        shutil.copytree(os.path.join(release_dir, "rules"), os.path.join(local_repo, kibana_directory))
+        git("add", kibana_directory)
+
+        git("commit", "-S", "-m", message)
+        git("status", show_output=True)
+
+        click.echo(f"Kibana repository {local_repo} prepped. Push changes when ready")
+        click.secho(f"cd {local_repo}", bold=True)
```

Comment:
Did you mean for this to print or execute?

Reply:
Yeah, my thought was that it makes copy and paste easier so you can go to that dir easier to run git commands before pushing